compiler/executable/match_/planner/plan.rs

```diff
@@ -832,6 +836,7 @@ impl ConjunctionPlan<'_> {
                 let negation = negation.plan().lower(
                     match_builder.current_outputs.iter().copied(),
                     match_builder.position_mapping(),
+                    &match_builder.position_mapping().keys().copied().collect(),
```

Comment:
@dmitrii-ubskii : This is equivalent to what it was before the change.
 Is this ok, or should it be: 
```rust 
&negation.variables().map(|id| self.graph.index_to_variable.get(&id).unwrap().clone()).collect_vec()```

Reply:
I think that ends up the same semantically. I'd expect `.keys().collect()` to be a little faster though.